typescript-light-bulb/main.ts

```diff
@@ -0,0 +1,18 @@
+const $bulb = document.querySelector('.bulb');
+const $background = document.querySelector('.background');
+let lightOn = false;
+if (!$bulb) throw new Error('Someone had better hurry over to Home Depot!');
+$bulb.addEventListener('click', clickHandler);
+function clickHandler(): void {
+  if (!$bulb) throw new Error('Someone had better hurry over to Home Depot!');
+  if (!$background) throw new Error("You're off the map!");
+  if (lightOn === false) {
+    lightOn = true;
+    $bulb.className = 'bulb bulb-on';
```

Comment:
No change needed on this now, but you could can shorten this with _truthy_/_falsy_ evaluation. Instead of `if (lightOn === false)` you could just go with `if (!lightOn)` which evaluates if `lightOn` is not `true`.